src/components/RKSignersModal.tsx

```diff
@@ -22,8 +22,12 @@ import { healthCheckStatusUpdate } from 'src/store/sagaActions/bhr';
 import { hcStatusType } from 'src/models/interfaces/HeathCheckTypes';
 import { getTxHexFromKeystonePSBT } from 'src/hardware/keystone';
 
-const RKSignersModal = ({ data }, ref) => {
-  const { serializedPSBTEnvelop, signer, vault, vaultId, vaultKey, isMultisig } = data;
```

Comment:
vaultID and vaultKey are still used in various places in the screens. See errors in lines 120 - 123, 137 - 145, 169 - 177, 180 - 185, 197 - 205, 229 - 237.

Reply:
Removed the whole sign fn for now, will be implementing along with each signer addition to Remote key flow.